### PR TITLE
Bc1 25 features section show land

### PR DESCRIPTION
**Pull request by sadiGFG**

### CODE REVIEW

File: show-land/sass/layout/_page-builder.scss [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    width: 1.7rem;
+    height: 1.7rem;
+    border-radius: 10rem;
+    margin-bottom: -0.4rem;
+  }
+  &__button {
+    display: inline-flex;
+    align-items: baseline;
+    border-radius: 10rem;
+    padding: 0 1rem 0 1rem;
```
Comment (sinaalimi) [inline show-land/sass/layout/_page-builder.scss:19]: I suggest you write like this:
padding : 0 1rem :)

**Review by sinaalimi — COMMENTED** [review]
Please check the responsiveness of the page.
 You have a problem with the page's responsiveness. 
Thanks my friend.

**Comment by abdolmajed3** [comment]
please resolve the conflicts

File: show-land/sass/layout/_page-builder.scss (hunk 1 above)
Comment (abdolmajed3) [reply show-land/sass/layout/_page-builder.scss:19]: and you can use our variables. ;)
like this :
padding: list.nth($spaces, 4) 0

File: show-land/sass/layout/_features.scss [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  &__headline {
+    text-align: center;
+    font-size: 2.8rem;
+    font-weight: $fontWeightBold;
+    background-image: linear-gradient(to right, $colorEighth, $colorSeventh);
+    -webkit-background-clip: text;
+    color: transparent;
+    &_1 {
+      text-align: center;
+      font-size: 2.8rem;
```
Comment (ahmadmohammadirad2006) [inline show-land/sass/layout/_features.scss:24]: please use variables from shared folder

**Review by ahmadmohammadirad2006 — APPROVED** [review]

**Pull request merged**